tests/unit/backend/test_common.py

```diff
@@ -56,15 +56,17 @@ def test_default_empty_list_include(self):
 
     def test_string_include(self):
         """'PyRDF.include' with a single string."""
-        PyRDF.include("header1")
+        PyRDF.include("tests/unit/backend/test_headers/header1.hxx")
```

Comment:
This is fine for now, but I think we need to extend this `include` functionality to accept directories as well. Or perhaps add a new function to the API.

Reply:
Good point, opened an [issue](https://github.com/JavierCVilla/PyRDF/issues/19) to keep track of this.